wgpu-hal/src/metal/adapter.rs

```diff
@@ -202,7 +203,7 @@ impl crate::Adapter<super::Api> for super::Adapter {
                 } else if pc.msaa_desktop {
                     Tfc::SAMPLED_LINEAR
                 } else {
-                    Tfc::STORAGE
+                    Tfc::SAMPLED_LINEAR
```

Comment:
seems unrelated to the change, could we move it to a separate PR please?

Reply:
Ok :)